CONTRIBUTING.md

```diff
@@ -8,8 +8,8 @@ Architecture decisions for this project [are documented here][adrs], using the [
 
 > Pre-requisite: you have installed [git][install-git], [node][install-node] and [yarn][install-yarn].
 
-1. Clone the repo: `git clone git@github.com:nicoespeon/slides.git`
-1. Go into the cloned repository: `cd slides`
+1. Clone the repo: `git clone https://github.com/nicoespeon/vscode-slides.git`
```

Comment:
Out of curiosity: why the HTTPS protocol over the git one?

I'd leave the git one by default. In my experience, the HTTPS one might ask for credentials while the git protocol uses your SSH key.